docs/csharp/language-reference/keywords/goto.md

```diff
@@ -9,28 +9,33 @@ helpviewer_keywords:
 ms.assetid: 2c03c9c1-8119-44ef-b740-fb3d287a42fe
 ---
 # goto (C# Reference)
-The `goto` statement transfers the program control directly to a labeled statement.  
-  
- A common use of `goto` is to transfer control to a specific switch-case label or the default label in a `switch` statement.  
-  
- The `goto` statement is also useful to get out of deeply nested loops.  
-  
-## Example  
- The following example demonstrates using `goto` in a [switch](../../../csharp/language-reference/keywords/switch.md) statement.  
-  
- [!code-csharp[csrefKeywordsJump#4](../../../csharp/language-reference/keywords/codesnippet/CSharp/goto_1.cs)]  
-  
-## Example  
- The following example demonstrates using `goto` to break out from nested loops.  
-  
- [!code-csharp[csrefKeywordsJump#5](../../../csharp/language-reference/keywords/codesnippet/CSharp/goto_2.cs)]  
-  
-## C# Language Specification  
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
-  
-## See Also  
- [C# Reference](../../../csharp/language-reference/index.md)  
- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
- [C# Keywords](../../../csharp/language-reference/keywords/index.md)  
- [goto Statement](/cpp/cpp/goto-statement-cpp)  
- [Jump Statements](../../../csharp/language-reference/keywords/jump-statements.md)
+
+The `goto` statement transfers the program control directly to a labeled statement.
+
+A common use of `goto` is to transfer control to a specific switch-case label or the default label in a `switch` statement.
+
+The `goto` statement is also useful to get out of deeply nested loops.
+
+## Example
+
+The following example demonstrates using `goto` in a [switch](switch.md) statement.
+
+[!code-csharp[csrefKeywordsJump#4](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsJump/CS/csrefKeywordsJump.cs#4)]
+
+## Example
+
+The following example demonstrates using `goto` to break out from nested loops.
+
+[!code-csharp[csrefKeywordsJump#5](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsJump/CS/csrefKeywordsJump.cs#5)]
+
+## C# language specification
+
+[!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]
+
+## See also
+
+[C# Reference](../index.md)  
+[C# Programming Guide](../../programming-guide/index.md)  
+[C# Keywords](index.md)  
+[goto Statement](/cpp/cpp/goto-statement-cpp)  
```

Comment:
Is this right? a cpp link instead of C#?

Reply:
I didn't change this. But given you're already on the goto C# page, it's probably right. Might help to add a suffix there or just remove the link. @BillWagner any preference?